docs/common-issues/clipped-text.md

```diff
@@ -0,0 +1,49 @@
+# Issue: Text is clipped [Draft]
+
+Long string of text not breaking to accommodate the width of the parent container. The text gets clipped.
```

Comment:
I'd suggest this more conversational and describe other possible issues / causes
eg "Text is present which does not wrap to stay within the bounds of its parent container. This is causing the text to either extend beyond the container, causing the layout to break or scrollbars to appear, or the text is clipped by the parent container and content is lost."

Reply:
Might be worth making the distinction between this happening because of text-resize and because of poorly implemented responsive design (Reflow).